### ISSUE TITLE
Unable to review terminal output with NVDA in focus mode.

### ISSUE THREAD

**Issue by TheQuinbox** [microsoft/vscode#172658]
Type: <b>Bug</b>

STR:
1. Start NVDA
2. Open a terminal with control+`
3. Run a few commands to generate output.
4. Try to navigate the output with control+up and down.

I hear list item, list item, list item. To actually read the text of each item, I have to hit NVDA space, read, then hit NVDA space again.
I don't remember exactly when, I think in the 1.6x days, this used to work, but it hasn't for quite a while now.

VS Code version: Code 1.74.3 (97dec172d3256f8ca4bfb2143f3f76b503ca0534, 2023-01-09T16:59:02.252Z)
OS version: Windows_NT x64 10.0.22000
Modes:
Sandboxed: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 7 5800H with Radeon Graphics          (16 x 3194)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|undefined|
|Memory (System)|15.41GB (9.47GB free)|
|Process Argv|--crash-reporter-id 5211d001-3acc-473c-a716-32f72b6395d5|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (9)</summary>

Extension|Author (truncated)|Version
---|---|---
copilot|Git|1.70.8099
isort|ms-|2022.8.0
python|ms-|2022.20.2
vscode-pylance|ms-|2023.1.40
jupyter|ms-|2022.11.1003412109
jupyter-keymap|ms-|1.0.0
jupyter-renderers|ms-|1.0.14
vscode-jupyter-cell-tags|ms-|0.1.6
vscode-jupyter-slideshow|ms-|0.1.5

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492:30256859
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242:30382549
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263cf:30335440
vscorecescf:30445987
pythondataviewer:30285071
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593:30376534
pythonvs932:30410667
cppdebug:30492333
vsclangdc:30486549
c4g48928:30535728
dsvsc012:30540252
azure-dev_surveyone:30548225
pyindex848:30577860
nodejswelcome1cf:30587006
2e4cg342:30602488
f6dab269:30613381
2i9eh265:30646982
vscup:30628656

```

</details>

**Comment by meganrogge** [comment]
We can track navigation mode, what you're describing, not working in this issue. We're aware of it and hoping that our new terminal accessibility buffer offers a better solution over time so we can remove this all together.

**Comment by meganrogge** [comment]
related #170988
